tests/unit/test_split.py

```diff
@@ -6,7 +6,7 @@
 
 
 def test_ignore_go_inside_quoted_text():
-    seed = """DECLARE @td            VARCHAR(20)   = CONVERT(VARCHAR, GETDATE(), 112);
+    seed = """DECLARE @td            VARCHAR(20)   = CONVERT(VARCHAR, GETDATE(), 112)
```

Comment:
Thanks to your comment, I could follow that the change in `test_dynamics` is due to https://github.com/antlr/grammars-v4/issues/3275. 

It's not clear to me why this change - and all other changes removing semicolons - is necessary. Was the previous grammar too lax; do we actually think this is an illegitimate statement? Or is this an unfortunate confinement of the newly proposed grammar?